src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -278,21 +226,33 @@
       "type": "computed",
       "value": "(auth == \"Individual\")"
     },
-    "IndividualAuth": {
-      "type": "computed",
-      "value": "(auth == \"Individual\" || auth == \"IndividualB2C\")"
-    },
     "IndividualB2CAuth": {
       "type": "computed",
       "value": "(auth == \"IndividualB2C\")"
```

Comment:
internalAuth

Reply:
You also need to update SingleOrgAuth and MultiOrgAuth computed symbols as well, like was done with Razor template